apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -273,28 +273,116 @@ const VerticalOverflow = () => {
 const HorizontalOverflow = () => {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 22 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26730&commitId=d04deccbbca9095e7205c7583fa28040ad25101a&env=prod)

✅ There was 5 screenshots added, 0 screenshots removed, 1820 screenshots unchanged, 0 screenshots with different dimensions and 17 screenshots with visible difference.


  <details><summary><b>unknown</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.size.rest.chromium.png|778| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|39| Changed|
|Positioning.arrow - RTL.chromium.png|71| Changed|
|Positioning.arrow.chromium.png|72| Changed|
|Positioning.auto size.chromium.png|261| Changed|
|Positioning.horizontal overflow - RTL.chromium.png|2073| Changed|
|Positioning.horizontal overflow.chromium.png|2073| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|193| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|331| Changed|
|Tooltip Converged.basic.chromium.png|203| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|1| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|6| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|370| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|160| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|160| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|17| Changed|
|Tooltip Converged.withArrow.chromium.png|1| Changed|
|Positioning.explicit overflow padding - RTL.chromium.png|0| Added|
|Positioning.explicit overflow padding.chromium.png|0| Added|
|Positioning.horizontal overflow padding - RTL.chromium.png|0| Added|
|Positioning.horizontal overflow padding.chromium.png|0| Added|
|Positioning.vertical overflow padding.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
The screenshots are the result of the following PRs which change  the way rects are calculated in Floating UI
* https://github.com/floating-ui/floating-ui/pull/2056 
* https://github.com/floating-ui/floating-ui/pull/2019